imsv-docs-docusaurus/openapi/endpoints/funding-sources/list-funding-sources.yaml

```diff
@@ -3,11 +3,11 @@ get:
     - funding-source
   summary: List Funding Sources
   operationId: list-funding-sources
-  description: List Funding Sources associated with the cardholder account
+  description: List Funding Sources associated with the account
   parameters:
     - name: accountId
       in: path
-      description: Cardholder account ID to retrieve Funding Sources for.
+      description: Account ID to retrieve Funding Sources for.
```

Comment:
this is a duplicate with my PR: https://github.com/immersve/immersve-docs/pull/544

Reply:
im going to resolve this, because it doesn't matter.